api/features/tasks.py

```diff
@@ -50,6 +57,39 @@ def trigger_feature_state_change_webhooks(
         )
     )
 
+    if hasattr(instance.environment.project.organisation, "github_config"):
+        github_configuration = instance.environment.project.organisation.github_config
+
+        feature_state = {
+            "environment_name": new_state["environment"]["name"],
+            "feature_value": new_state["enabled"],
+        }
+        feature_states = []
+        feature_states.append(instance)
+
+        feature_data = generate_data(
+            github_configuration=github_configuration,
+            feature_id=history_instance.feature.id,
+            feature_name=history_instance.feature.name,
+            type=WebhookEventType.FLAG_UPDATED,
+            feature_states=feature_states,
+        )
+
+        feature_data["feature_states"].append(feature_state)
+
+        call_github_app_webhook_for_feature_state.delay(
+            args=(
+                feature_data,
+                event_type.value,
+            ),
+        )
+    else:
+        logger.warning(
```

Comment:
I don't think there is any use for this warning?

Reply:
Perhaps while the PR isn't approved, this logger can provide us with additional information.